frontend/test/metabase/scenarios/question/notebook.cy.spec.js

```diff
@@ -21,418 +21,444 @@ const { ORDERS, ORDERS_ID, PEOPLE, PEOPLE_ID, PRODUCTS, PRODUCTS_ID } =
+        dataset: true,
+        display: "table",
+      });
+
+      startNewQuestion();
+      popover().findByText("Models").click();
+      popover().findByText("Products model").click();
+      join();
+      popover().findByText("Orders model").click();
+      popover().findByText("ID").click();
+      popover().findByText("Product ID").click();
+
+      visualize();
     });
 
-    startNewQuestion();
-    popover().findByText("Models").click();
-    popover().findByText("Products model").click();
-    join();
-    popover().findByText("Orders model").click();
-    popover().findByText("ID").click();
-    popover().findByText("Product ID").click();
+    // flaky test
+    it.skip("should show an info popover when hovering over a field picker option for a table", () => {
+      startNewQuestion();
+      cy.contains("Sample Database").click();
+      cy.contains("Orders").click();
 
-    visualize();
-  });
+      cy.findByTestId("fields-picker").click();
 
-  // flaky test
-  it.skip("should show an info popover when hovering over a field picker option for a table", () => {
-    startNewQuestion();
-    cy.contains("Sample Database").click();
-    cy.contains("Orders").click();
+      cy.findByText("Total").trigger("mouseenter");
 
-    cy.findByTestId("fields-picker").click();
+      popover().contains("The total billed amount.");
+      popover().contains("80.36");
+    });
 
-    cy.findByText("Total").trigger("mouseenter");
+    // flaky test
+    it.skip("should show an info popover when hovering over a field picker option for a saved question", () => {
+      cy.createNativeQuestion({
+        name: "question a",
+        native: { query: "select 'foo' as a_column" },
+      });
 
-    popover().contains("The total billed amount.");
-    popover().contains("80.36");
+      // start a custom question with question a
+      startNewQuestion();
+      cy.findByText("Saved Questions").click();
+      cy.findByText("question a").click();
+
+      cy.findByTestId("fields-picker").click();
+
+      cy.findByText("A_COLUMN").trigger("mouseenter");
+
+      popover().contains("A_COLUMN");
+      popover().contains("No description");
+    });
   });
 
-  // flaky test
-  it.skip("should show an info popover when hovering over a field picker option for a saved question", () => {
-    cy.createNativeQuestion({
-      name: "question a",
-      native: { query: "select 'foo' as a_column" },
+  describe("postgres", () => {
+    beforeEach(() => {
+      restore("postgres-12");
+      cy.signInAsAdmin();
     });
 
-    // start a custom question with question a
-    startNewQuestion();
-    cy.findByText("Saved Questions").click();
-    cy.findByText("question a").click();
+    it('should show Median aggregation function for databases that support "percentile-aggregations" driver feature', () => {
+      // start a custom question with orders
+      startNewQuestion();
+      cy.contains("QA Postgres12").click();
```

Comment:
We prefer `testing-library` commands, like `cy.findByText("QA Postgres12").click()`.

Reply:
When interacting with something in a popover, we use `popover` helper like this:
```
popover().within(() => {
  cy.findByText("QA Postgres12").click();
  cy.findByText("Products").click();
});
```